.buildkite/pipeline.yml

```diff
@@ -1,231 +0,0 @@
-# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
-# SPDX-License-Identifier: Apache-2.0
```

Comment:
We will need to keep this file so that we are not forced to update all rust-vmm components at once. Once we roll out the new rust-vmm-ci to all components, we can add a PR to delete this file.